recipes/fastpdb/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: fastpdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/fastpdb/archive/v{{ version }}.tar.gz
+  sha256: 1cad47ebdbd969be772f64a8900738b94899f664b726ea520dab4ef14a65a5a8
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py < 310]
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - numpy >=2.0                            # [build_platform != target_platform]
+    - {{ stdlib("c") }}
+    - {{ compiler("rust") }}
+  host:
+    - python
+    - maturin >=1.4
+    - numpy >=2.0
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```

Comment:
Please don't use this anymore. Numpy has a run_exports.
```suggestion
```

Reply:
Thanks, I accidentally took this over from an older recipe.